frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -23,6 +25,13 @@ const normalizePackageManifests = (packageManifests, kind) => {
     const tileDescription = packageManifest.metadata.description;
     const tileProvider = _.get(packageManifest, 'metadata.labels.provider');
     const tags = packageManifest.metadata.tags;
+    const certifiedLevel = packageManifest.metadata.certifiedLevel;
+    const version = _.get(packageManifest, 'status.channels[0].currentCSVDesc.version');
+    const healthIndex = packageManifest.metadata.healthIndex;
+    const repository = packageManifest.metadata.repository;
+    const containerImage = packageManifest.metadata.containerImage;
+    const createdAt = packageManifest.metadata.createdAt;
+    const support = packageManifest.metadata.support;
```

Comment:
```
const { tags, certifiedLevel, version, healthIndex, repository, containerImage, createdAt, support } = packageManifest.metadata;
```

renaming the const `tileDescription` to `description` would allow you to add it to ^^